CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+### Added
+- OE SDK Release packages can now be built on non SGX and non FLC machines.
+
+### Changed
+
+### Removed
+- HAS_QUOTE_PROVIDER is deprecated as a cmake option. It has not effect on SDK build.
```

Comment:
I'm not entirely sure this needs deprecation ... have we exposed this flag for public use by the developer? If it's just an internal build flag, we can just get rid of it.

Also typo "not" -> "no"

Reply:
Currently this flag is exposed to the SDK contributor and documented in all the SGX1 (non FLC) docs
E.g: https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md

Hence it seems appropriate to deprecate it.

Reply:
Fixed typo.

Reply:
Removed HAS_QUOTE_PROVIDER instead of deprecated based on discussion in this PR.